sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/samples/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStoreSample.java

```diff
@@ -11,91 +12,99 @@
 import redis.clients.jedis.JedisPool;
 
 import java.time.Duration;
+import java.time.Instant;
 import java.util.Collections;
 import java.util.List;
-import java.util.StringJoiner;
+import java.util.UUID;
 
 /**
  * Sample that demonstrates the use of {@link JedisRedisCheckpointStoreSample} for storing and updating partition
- * ownership records using Redis cache.
+ * ownership records in Azure Redis Cache.
  */
 public class JedisRedisCheckpointStoreSample {
-    private static final String FULLY_QUALIFIED_NAMESPACE = "example-eh-namespace.servicebus.windows.net";
-    private static final String EVENT_HUB_NAME = "my-event-hub";
-    private static final String CONSUMER_GROUP = "my-consumer-group";
-
-    /**
-     * Maximum time to wait for operation to complete.
-     */
-    private static final Duration TIMEOUT = Duration.ofMinutes(2);
+    private static final String EVENT_HUB_NAMESPACE = "{your-namespace}.servicebus.windows.net}";
```

Comment:
nit; Do we need the last curly bracket? -> "....windows.net}"

Reply:
nope. thanks for the catch